includes/class-genesis-simple-faq-widget.php

```diff
@@ -63,19 +69,19 @@ public function widget( $args, $instance ) {
 			Genesis_Simple_FAQ()->assets->enqueue_scripts();
 		}
 
-		echo $args['before_widget'];
+		echo wp_kses_post( $args['before_widget'] );
```

Comment:
As we discussed in [AP Listings](https://github.com/studiopress/agentpress-listings/pull/31#issuecomment-490657840), not using `wp_kses_post` here is acceptable for
$before_widget
$after_widget
$before_title
$after_title